### ISSUE TITLE
"body" field for "Service Sub-landing page" content type disabled

### ISSUE THREAD

**Issue by bisd** [localgovdrupal/localgov_services#186]
Creating a new instance of the above content type results in the headline showing but not the body. So the user has to go to the “Manage display” page for that content type and manually enable the disabled by default "body" field.

**Comment by markconroy** [comment]
Editing the title to remove reference to the `localgov_base` theme, as this is purely a site builder/config issue, not a theme issue.

**Comment by MichaelDCR** [comment]
This is in the wrong Repo

**Comment by bisd** [comment]
Thank you MichaelDCR would you like me to move it to a different repo?

**Comment by keelanfh** [comment]
I think this is confusing on a few content types:

* service landing page
* service sub-landing page
* guide overview

If my understanding is correct, the 'body' field appears on all these content types, but it doesn't show up in the actual site. The reason that the body field is there is so that a truncated version of the body (or the summary, if present) is used as teaser text on the parent page of these pages. So it's called 'body' and supports rich text styling, but never actually appears as such.

We could label this field as 'teaser text' and reduce to one field (rather than having a summary and body field) - example of how we do this in our existing CMS below. But this would cause problems for councils that have changed from the default configuration to actually show the body text on their site.

![image](https://user-images.githubusercontent.com/19519457/213778342-2818d5e2-0c9c-4ffa-95c6-6e6c34113086.png)

**Comment by markconroy** [comment]
I find the body field quite handy on this content types, so make it display on the page, rather than using it as "just" a teaser field.

**Comment by keelanfh** [comment]
Yeah, it's a bit of a tricky one - I can see that it needs to be there for councils that have the field visible, but I think in the default setup it can be confusing...

**Comment by finnlewis** [comment]
One for the forthcoming product group /proudct team to discuss?
